### ISSUE TITLE
nodejs distributions are not affected the same way by security releases

### ISSUE THREAD

**Issue by kapouer** [nodejs/security-wg#786]
Hi,
as an example, in Debian we have Node.js depending on system-installed OpenSSL,
thus security issues affecting that dependency are, most of the time, dealt with in the corresponding dependency, not in Node.js.

Thus a sentence like "if you're using a Node.js version not distributed by official upstream channel, this security issue might not concern you, please check with their security team", etc...

**Comment by DanielRuf** [comment]
Hi @kapouer,

I'm not sure I fully understand what you mean. Do you mean an improvement for security release notes? Isn't this handled by the release WG? What is the current sentence like and where can we see it?

https://github.com/nodejs/release

They also define what gets backported:

> Define the policy for what gets backported to release streams.

Sure, most distros and the package maintainers there who package the software for the target distros most often recompile, repackage and backport upstream fixes.

**Comment by mhdawson** [comment]
That is true for RHEL distributions as well.  I'm going to move this into the TSC repo for discussion to see what we  might be willing to add to the pre-release announce.
